crates/frontend/src/components/experiment.rs

```diff
@@ -162,62 +187,52 @@ where
                             for token in contexts.clone() {
                                 let (dimension, values) = (token.left_operand, token.right_operand);
                                 let mut value_views = Vec::new();
-
                                 for value in values.iter() {
                                     if value.is_object() && value.get("var").is_some() {
                                         continue;
                                     }
-
                                     let value_str = match value {
                                         Value::String(s) => s.clone(),
                                         Value::Number(n) => n.to_string(),
                                         Value::Bool(b) => b.to_string(),
                                         Value::Null => String::from("null"),
                                         _ => format!("{}", value),
                                     };
-
-                                    value_views.push(
-                                        view! {
+                                    value_views
+                                        .push(
+                                            view! {
                                                 <div class="stat-value text-base">
                                                     {&value_str.replace('"', "")}
                                                 </div>
-                                        },
-                                    );
-
+                                            },
+                                        );
                                 }
-                                view.push(view! {
-                                    <div class="stat w-3/12">
-                                        <div class="stat-title">{dimension}</div>
-                                        {value_views}
-                                    </div>
-                                });
-
+                                view.push(
+                                    view! {
+                                        <div class="stat w-3/12">
+                                            <div class="stat-title">{dimension}</div>
+                                            {value_views}
+                                        </div>
+                                    },
+                                );
                             }
                             view
                         }}
 
                     </div>
                 </div>
-            </div> <div class="card bg-base-100 max-w-screen shadow m-5">
+            </div>
+            <div class="card bg-base-100 max-w-screen shadow m-5">
                 <div class="card-body">
```

Comment:
meanwhile can we also remove this redundant `div`

Reply:
Sure